test/e2e/targets/azureeventhubs/main.go

```diff
@@ -276,3 +375,11 @@ func withEventHubID(subscriptionID, resourceGroup, eventHubNS, eventHub string)
 		}
 	}
 }
+
+func withoutCloudEvents() targetOption {
```

Comment:
I'm sure we can come up with a better name, e.g.

https://github.com/triggermesh/triggermesh/blob/fb11bdeb11dbb3d1ce38c07a646886fd5cb55d43/test/e2e/targets/awss3/main.go#L327